environments/azure/flyte-core/README.md

```diff
@@ -118,6 +118,100 @@ You have a fully working Flyte environment on Azure.
 
 From this point on, you can continue your learning journey by going through the [Flyte Fundamentals](https://docs.flyte.org/en/latest/flyte_fundamentals/index.html) tutorials.
 
+## Consuming GPU accelerators
+
+To be able to request GPUs on Azure directly from your Flyte tasks, you have two main options. This section covers both and how to use them with the Terraform/OpenTofu modules:
```

Comment:
What exactly you mean with two main options? In this ReadMe you are following up with 1,2,3